BricklinkSharp.Client/BricklinkClient.cs

```diff
@@ -734,6 +776,7 @@ public void Dispose()
 
         var url = new Uri(_baseUri, $"coupons/{couponId}").ToString();
 
+        _measureRequest(cancellationToken);
```

Comment:
MeasureRequest should be awaited?
```suggestion
        await MeasureRequestAsync(cancellationToken);
```